<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ordersWindow</class>
 <widget class="QMainWindow" name="ordersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #fff;
}
QPushButton {
    border: 2px solid gray;
    border-radius: 6px;
    min-width: 80px;
	font: 13pt &quot;Liberation Serif&quot;;
	color: black;
	background-color: white
}
QPushButton:pressed {
    border: 2px solid blue;
}
QComboBox {
    border: 2px solid gray;
    border-radius: 6px;
    min-width: 80px;
	font: 13pt &quot;Liberation Serif&quot;;
	color: black;
	background-color: white;
}
QLineEdit {
	color: black;
	background-color: #ddd;
 	border-radius: 8px;
 	border: 1px solid #e0e4e7;
	padding: 5px 15px;
}
QTableWidget {
	color: black;
}
QLabel {
	color: black;
	font: 11pt &quot;DejaVu Math TeX Gyre&quot;;
}
QMenu {
	color: black;
	border: 2px solid gray;
	border-radius: 12px 12px 0px 0px;
    min-width: 80px;
}
QMenuBar {
	color: black;
	border: 2px solid gray;
    min-width: 80px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>81</y>
      <width>721</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ID заявки</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enterID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>271</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="approveOrder">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Одобрить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rejectOrder">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Отклонить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveTable">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>380</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Экспорт таблицы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 450 13pt &quot;Fira Code&quot;;</string>
    </property>
    <property name="text">
     <string>Все открытые вклады</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="exitFromAccount">
   <property name="text">
    <string>Выйти из аккаунта</string>
   </property>
  </action>
  <action name="exitMenu">
   <property name="text">
    <string>Выйти в меню</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
